hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMRatisServerImpl.java

```diff
@@ -208,12 +267,12 @@ public RaftPeerId getPeerId() {
         String host = hosts.get(i);
         if (InetAddress.getByName(host).equals(localHost)) {
           selfPeerId = peerId;
+          raftPeers.add(RaftPeer.newBuilder()
+              .setId(peerId)
+              .setAddress(host + ":" + port)
+              .build());
+          break;
```

Comment:
why need this change ? `raftPeers` should contain all the SCMs in `ozone.scm.names`. With it, we can not build a 3 SCM cluster any more.